src/backend/aspen/api/views/groups.py

```diff
@@ -62,17 +64,15 @@ async def get_group_info(
     group_id: int,
     request: Request,
     db: AsyncSession = Depends(get_db),
-    user: User = Depends(get_auth_user),
+    authorized_query: Select = Depends(require_access("read", Group)),
```

Comment:
ooooh.... doesn't have to happen right now, but maybe we have a dependency that finds the group(/[object]) by the id referenced in the endpoint path automatically!!